controller_manager/test/controller_manager_test_common.hpp

```diff
@@ -0,0 +1,83 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
if this is a completely new file, you may want to adjust the license

---> also in general please make sure licenses are applied where due

Reply:
let's leave it as-is then